src/forms/RadioButtonGroup.jsx

```diff
@@ -1,73 +1,68 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 
-import Flex from '../layout/Flex';
-import FlexItem from '../layout/FlexItem';
-
 /**
  * @class RadioButtonGroup
  */
 export default class RadioButtonGroup extends PureComponent {
 	constructor(props) {
 		super(props);
-		this.state = {
-			selectedValue: props.selectedValue,
+
+		const initialState = {
+			value: props.selectedValue,
 		};
-		this.handleChange = this.handleChange.bind(this);
+
+		React.Children.forEach(props.children, radio => {
+			initialState[radio.props.value] = radio.props.value === props.selectedValue;
```

Comment:
I might be missing it, but it doesn't look like this 'radio value' state is being used at all in rendering? I see it being written here and in `componentWillReceiveProps`, but never being read. Can you go back to just having `this.state.selectedValue`?